Sources/GRPC/AsyncAwaitSupport/GRPCSendable.swift

```diff
@@ -13,25 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import NIOCore
 
-#if compiler(>=5.6)
+@available(*, deprecated, renamed: "Swift.Sendable")
 public typealias GRPCSendable = Swift.Sendable
-#else
-public typealias GRPCSendable = Any
-#endif // compiler(>=5.6)
 
-#if compiler(>=5.6)
+@available(*, deprecated, message: "Use @preconcurrency Swift.Sendable")
 @preconcurrency
 public protocol GRPCPreconcurrencySendable: Sendable {}
```

Comment:
We should deprecate this but we can't remove the inheritance from the types sadly since this is a `public protocol` and users might have done weird things based off that inheritance

Reply:
We can't deprecate this without warnings in our own code but I've switched back to using it again.